lldb/include/lldb/Core/Mangled.h

```diff
@@ -308,6 +308,75 @@ class Mangled {
 
 Stream &operator<<(Stream &s, const Mangled &obj);
 
+using namespace swift::Demangle;
+class TrackingNodePrinter : public NodePrinter {
```

Comment:
Can you add a doxygen comment explaining the purpose of this?
It's a layering violation to have a Swift-specific declaration inside of Core/Mangled.h
This declaration should probably live somewhere under Plugins/Language(Runtime)/Swift/